log4j2.xml

```diff
@@ -16,7 +143,25 @@
 
         <Logger name="org.springframework.web" level="${env:LOG_LEVEL_SPRING_WEB:-OFF}"/>
 
-        <Logger name="org.labkey" level="${env:LOG_LEVEL_LABKEY_DEFAULT:-FATAL}"/>
+        <!-- category for server side script messages -->
+        <Logger name="org.labkey.api.script.ScriptService.Console" level="${env:LOG_LEVEL_API_SETTINGS:-INFO}">
```

Comment:
I'm not familiar with the syntax for the `level` here, but I can imagine what it means.